package.json

```diff
@@ -59,8 +59,8 @@
         "@angular/router": "13.3.2",
         "@ngx-translate/core": "14.0.0",
         "@ngx-translate/http-loader": "7.0.0",
-        "@ngxs/logger-plugin": "3.7.3",
-        "@ngxs/store": "3.7.3",
+        "@ngxs/logger-plugin": "^3.8.0",
+        "@ngxs/store": "^3.8.0",
```

Comment:
please, pin versions 

```json
"@ngxs/logger-plugin": "3.8.0",
"@ngxs/store": "3.8.0",
```

and rebuild lock file

Reply:
done